_posts/2019-01-17-things-to-do-in-nashville.md

```diff
@@ -35,3 +35,5 @@ As a start to beginning your Nashville experience, let us suggest just a couple
 * High Garden: Rated one of the best tea houses in the country
 * The Cupcake Collection: A Germantown instution for almost homemade, classic Southern cupcakes
 * Dozen Bakery: A small bakeshop in the Wedgewood Houston area of town with French classic pastries
+
+Have more Nashville greats we need to add to our list? Did you visit one of our suggestions and want to tell us about it? Tweet us @PyTennessee!
```

Comment:
Add something about tweeting at us if they'd like more/different recommendations, and make `@PyTennessee` a link to https://www.twitter.com/PyTennessee

Reply:
Aye, will do!